### ISSUE TITLE
Vulkan instead of OpenGL

### ISSUE THREAD

**Issue by ghost** [Technici4n/rust-voxel-game#1]
Since you are beginning from scratch, it would be better to target Vulkan instead of OpenGL You could benefit from performance and cross-platform. You could use gfx[WIP] for this or even vulkano.

I don't think finding a name should be priority.

Rayon could be a good alternative for scripting language.

I quite like the idea of exploring game development in rust.

**Comment by ghost** [comment]
Also use of Amethyst could be a viable option.

**Comment by Technici4n** [comment]
I'm currently using OpenGL with gfx and glutin.
As you said, gfx is under heavy development to support Vulkan. Once that's done, I might consider switching to a different graphics API (like Vulkan), which wouldn't require much changes anyways.
I chose OpenGL because I already have C++ experience with it, because it's portable and because it still can be very fast.

I've decided not to use any game engine because I believe it's not worth it for such simple rendering.

You probably meant Rhai instead of Rayon ? The main problem would be speed, but LuaJIT interfacing also isn't that nice.
